PlayerSkeleton.java

```diff
@@ -35,7 +35,8 @@ private int tryPossibleMoves(State s, int[][] legalMoves, double[] weights) {
 				continue;
 			} 
 			//Step 2
-			double score = Heuristic.evaluate(ts, weights);
+			ts.putOrientAndSlot(orient, slot);
+			double score = HeuristicNew.evaluate(ts, weights);
 
```

Comment:
Related to new changes in TempState, note the setter method of piece orientation and slot (i.e. the column the piece is landing at). I could have opted to pass it down to heuristics class as an argument in the evaluate method but then it's too convoluted IMO.